solver/jobs.go

```diff
@@ -806,7 +808,7 @@ func (s *sharedOp) CacheMap(ctx context.Context, index int) (resp *cacheMapResp,
 		return nil, err
 	}
 	flightControlKey := fmt.Sprintf("cachemap-%d", index)
```

Comment:
non-blocking: we can remove this `flightControlKey` and just use `index` now, since we split up the flight control instances.